### PR TITLE
Add peaks svg

### LINKED ISSUE #14: Add peaks map code to pipeline

### PR DESCRIPTION
**Pull request by lindsayplatt**
Fixes #14. Merge #20 first (still have some data fixing to do with the fetch steps in `1_fetch.yml`). Not animated; only for one day. Also, poorly removes any path with an NA in it. Know a smarter way, but tabling for now.

To build, 

1. Be on VPN so that the s3_get will work. 
2. Run `scmake()` which should produce `3_visualize/out/anomaly_peaks.svg`:

image

### CODE REVIEW

File: 3_visualize/src/svg_utils_mapping.R [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    # TODO: borders still slightly wonky bc it doesn't line up with counties perfectly. 
+    `nags head` = list(
+      state = "north carolina",
+      counties = c(
+        "currituck"
+      )),
+    # This + simplifying to 0.5 took care of the weird line across NY
+    `staten island` = list(
+      state = "new york",
+      counties = c(
```
Comment (collnell) [inline 3_visualize/src/svg_utils_mapping.R:145]: Interesting. I've found success with CO also by first simplifying the polygons with `ms_simplify()` and then `st_buffer(0)`. I think the simplification makes the coordinates slightly more distinct where they don't intersect, and therefore respect their shapes a little better

**Review by collnell — APPROVED** [review]
👏🏼 👏🏼 👏🏼 👏🏼

File: 3_visualize/src/svg_utils_mapping.R (hunk 1 above)
Comment (lindsayplatt) [reply 3_visualize/src/svg_utils_mapping.R:145]: This is code I borrowed from the WBEEP water use map I made. I am using the spatial data included in the `maps` package, which is not exactly perfect. It gets dropped after applying the Albers Equal Area transform ... which is so weird.

```r
library(sf)
library(dplyr)

# It's there:
yes_co_sf <- maps::map("state", fill = TRUE, plot=FALSE) %>%
    sf::st_as_sf() %>%
    st_buffer(0)
plot(st_geometry(yes_co_sf ), col = "grey")

# Now it's not:
 proj_str <- "+proj=aea +lat_1=29.5 +lat_2=45.5 +lat_0=37.5 +lon_0=-96 +x_0=0 +y_0=0 +ellps=GRS80 +datum=NAD83 +units=m +no_defs"
no_co_sf <- maps::map("state", fill = TRUE, plot=FALSE) %>%
    sf::st_as_sf() %>%
    st_transform(proj_str) %>%
    st_buffer(0)
plot(st_geometry(no_co_sf), col = "grey")
```

Comment (lindsayplatt) [reply 3_visualize/src/svg_utils_mapping.R:145]: Pinpointing the problem a bit more ... it appears to only happen when I do `st_buffer(0)` AFTER transforming. If left out or done before the transform, Colorado sticks around.

```r
# Buffer after transform = no Colorado!
no_co_sf <- maps::map("state", fill = TRUE, plot=FALSE) %>%
    sf::st_as_sf() %>%
    st_transform(proj_str) %>% 
    st_buffer(0)
plot(st_geometry(no_co_sf), col = "grey")

# Buffer before transform = yay! there's CO
yay_co_sf <- maps::map("state", fill = TRUE, plot=FALSE) %>%
    sf::st_as_sf() %>% 
    st_buffer(0) %>%
    st_transform(proj_str)
plot(st_geometry(yay_co_sf), col = "grey")

# No buffer at all - yes, we still have CO!
yes_co_sf <- maps::map("state", fill = TRUE, plot=FALSE) %>%
    sf::st_as_sf() %>% 
    st_transform(proj_str)
plot(st_geometry(yes_co_sf), col = "grey")
```

Comment (lindsayplatt) [reply 3_visualize/src/svg_utils_mapping.R:145]: In the interest of time, I'm going to stick this info into an issue to circle back later.

Comment (lindsayplatt) [reply 3_visualize/src/svg_utils_mapping.R:145]: See #23

**Pull request merged**